src/main/java/uk/gov/hmcts/reform/sscs/jms/listener/HmcHearingsEventTopicListener.java

```diff
@@ -72,18 +80,24 @@ public void onMessage(JmsBytesMessage message) throws JMSException, HmcEventProc
                 String hearingId = hmcMessage.getHearingId();
 
                 log.info(
-                        "Attempting to process message from HMC hearings topic for event {}, Case ID {}, and Hearing ID {}.",
-                        hmcMessage.getHearingUpdate().getHmcStatus(),
-                        caseId,
-                        hearingId
+                    "Attempting to process message from HMC hearings topic for event {}, Case ID {}, and Hearing ID {}.",
+                    hmcMessage.getHearingUpdate().getHmcStatus(),
+                    caseId,
+                    hearingId
                 );
 
-                processHmcMessageService.processEventMessage(hmcMessage);
+                if (processEventMessageV2Enabled) {
```

Comment:
So this condition evaluation will not be required based on above change.

Reply:
done